packages/hint-compat-api/src/html.ts

```diff
@@ -86,11 +86,20 @@ export default class HTMLCompatHint implements IHint {
 
             const report = ({ feature, unsupported }: ReportData) => {
                 const message = getMessage('featureNotSupported', context.language, [feature, joinBrowsers(unsupported)]);
+                let documentation: ProblemDocumentation[] | undefined;
+
+                if (unsupported.mdnUrl) {
```

Comment:
Nit: you can use the same inline initialization to make this `const` here as I suggested for the CSS case.